instance/static/html/instance/details.html

```diff
@@ -71,9 +71,9 @@ <h3 ng-if="instance">
         <tr><td>Modified</td><td>{{ instance.modified | date:'yyyy-MM-dd HH:mm:ssZ' }}</td></tr>
         <tr><td>Type</td><td>{{ instance.instance_type }}</td></tr>
         <!-- The following are all specific to Open edX instances: -->
-        <tr><td>Domain</td><td>{{ instance.domain}}<br>Sub domain: {{ instance.sub_domain}}, base domain: {{ instance.base_domain }}</td></tr>
         <tr><td>LMS URL</td><td><a href="{{ instance.url}}">{{ instance.url}}</a></td></tr>
         <tr><td>Studio URL</td><td><a href="{{ instance.studio_url}}">{{ instance.studio_url}}</a></td></tr>
+        <tr><td>Internal domain</td><td>{{ instance.internal_lms_domain }}</td></tr>
```

Comment:
@itsjeyd We display URLs that will use external domains when present in two lines above. I think we need to show the internal domain somewhere so that the customer will know what to point their CNAME record to.